src/win/winapi.c

```diff
@@ -63,6 +64,10 @@ void uv_winapi_init(void) {
     uv_fatal_error(GetLastError(), "GetModuleHandleA");
   }
 
+  pRtlGetVersion = (sRtlGetVersion) GetProcAddress(
+    ntdll_module,
+    "RtlGetVersion");
+
```

Comment:
Style: indentation should be consistent in the file (when getting `RtlNtStatusToDosError` it seems we're using 4-space indentation and I don't know which is the correct one though)